### PR TITLE
speed up --show_owners via authd api

### PR DESCRIPTION
**Pull request by elv-todd**
fixes https://github.com/eluv-io/elv-live-js/issues/136

sample output:
```
$ ./elv-live nft_show 0x4b2e8e711fb223096c3c16d0a9e3d7b835fb5aaa --show_owners 2
NFT - show
addr  0x4b2e8e711fb223096c3c16d0a9e3d7b835fb5aaa
check_minter  undefined
show_owners  2
show_owners_via_contract  undefined
token_id  undefined
Network: demov3
name: GOAT-NFT-TEST-URI
owner: '0x761f45287Ea364DB6b216bd655910430afA3e839'
symbol: GOAT-NFT-TEST-URI
totalSupply: 32
transferFee: 0
minted: 44
cap: 100
proxy: '0xFd612AF562108610e456c51B2c40a1053e8aed33'
proxyInfo:
  owner: '0x25D7D6e2Dc74FD56A4eacfd842B68eCF73B91f9D'
defHoldSecs: '604800'
tokens:
  - ordinal: '43'
    holdSecs: 1691525593
    holdEnd: 2023-08-08T20:13:13.000Z
    tokenId: '44'
    tokenURI: >-
      https://demov3.net955210.contentfabric.io/s/demov3/q/hq__AqmrLPerNTtkQ62jGYSJyhoNEkvQWGCp6MxTsDstoQzGp4eqjvmfLUE1kKLaXoK5uFbfFHsUoH/meta/public/nft
    owner: '0xb516b92fe8f422555f0d04ef139c6a68fe57af08'
  - ordinal: '42'
    holdSecs: 1691525354
    holdEnd: 2023-08-08T20:09:14.000Z
    tokenId: '43'
    tokenURI: >-
      https://demov3.net955210.contentfabric.io/s/demov3/q/hq__AqmrLPerNTtkQ62jGYSJyhoNEkvQWGCp6MxTsDstoQzGp4eqjvmfLUE1kKLaXoK5uFbfFHsUoH/meta/public/nft
    owner: '0xb516b92fe8f422555f0d04ef139c6a68fe57af08'
firstTokenUri: >-
  https://demov3.net955210.contentfabric.io/s/demov3/q/hq__AqmrLPerNTtkQ62jGYSJyhoNEkvQWGCp6MxTsDstoQzGp4eqjvmfLUE1kKLaXoK5uFbfFHsUoH/meta/public/nft
warns: []
```

new usage:
```
 nft_show <addr> [options]

Show info on this NFT

Positionals:
  addr  NFT address (hex)                                    [string] [required]

Options:
      --version                   Show version number                  [boolean]
  -v, --verbose                   Verbose mode                         [boolean]
      --as_url                    Alternate authority service URL (include
                                  '/as/' route if necessary)            [string]
      --help                      Show help                            [boolean]
      --check_minter              Check that all NFTs use this mint helper
      --show_owners               Show up to these many owners (default 0),
                                  loaded from an index. Only used when token_id
                                  is not specified.
      --show_owners_via_contract  Show up to these many owners (default 0),
                                  parsed directly from the contract. Only used
                                  when token_id is not specified.
      --token_id                  External token ID. This will take precedence
                                  over show_owners.                     [string]
```

### CODE REVIEW

**Comment by elv-todd** [comment]
i'd be tempted to use proper json formats instead of maintaining the old ones but, didn't want to rock the boat

File: src/EluvioLive.js [hunk 1]
```diff
@@ -1347,9 +1348,63 @@ class EluvioLive {
...
+          // token_id -> tokenId
+          nftInfo.tokens[i].tokenId = "" + nftInfo.tokens[i].token_id;
+          // token_uri -> tokenURI
+          nftInfo.tokens[i].tokenURI = nftInfo.tokens[i].token_uri;
+          // token_owner -> owner
+          nftInfo.tokens[i].owner = nftInfo.tokens[i].token_owner;
+          // ordinal int -> string
+          nftInfo.tokens[i].ordinal = "" + nftInfo.tokens[i].ordinal;
+
+          delete nftInfo.tokens[i].block;
```
Comment (elv-todd) [inline src/EluvioLive.js:1385]: instead of setting everything and then deleting, can set one at a time from res.contents.  will change

**Review by elv-Loren — APPROVED** [review]
looks good.  did not have a chance to test run it.

**Review by elv-serban — APPROVED** [review]
Looks great!

**Pull request merged**